JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1 +1,13 @@
-The **DataGrid** widget allows you to&nbsp;customize its toolbar using the **onToolbarPreparing** function. You can add new items to&nbsp;the toolbar or&nbsp;customize the existing ones. In&nbsp;this demo, the toolbar contains a&nbsp;collection of&nbsp;other widgets from the DevExtreme HTML5&nbsp;UI Widgets library.
\ No newline at end of file
+The toolbar contains predefined and custom controls that execute various commands.
+
+The **DataGrid** uses the DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component as the toolbar. To customize the toolbar, implement the [onToolbarPreparing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onToolbarPreparing) function. Its parameter contains the [Toolbar's configuration](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/) in the **toolbarOptions** field. You can access toolbar items via the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/) array. Modify this array to add or remove toolbar items.
+
+This demo illustrates how to add the following elements to the toolbar:
+
+- **DevExtreme widgets**        
+Specify the [widget](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#widget) that you want to add and its [options](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#options). In this demo, we added the [Button](/Demos/WidgetsGallery/Demo/Button/PredefinedTypes/) and [SelectBox](/Demos/WidgetsGallery/Demo/SelectBox/Overview/) widgets.
+
+- **Custom elements**     
+Specify a [template](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#template) for your custom element. In this demo, the custom element displays the total group count.
+
+Toolbar items in this demo also include the Column Chooser button. This is a predefined button that the **DataGrid** added because the [columnChooser](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columnChooser/) is enabled.
```

Comment:
If other section have titles, I guess we should add a title for this paragraph - something like "Built-in commands".

Reply:
```suggestion
- **Predefined controls**        
The availability of predefined controls depends on whether a specific **DataGrid** feature is enabled. For example, this demo enables the [columnChooser](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columnChooser/), and the toolbar displays the Column Chooser button. You can customize the predefined controls in the **onToolbarPreparing** function. Refer to its [description]() for a code example.
```

?

Reply:
Looks good to me. Thanks